docs/pages/access-controls/guides/locking.mdx

```diff
@@ -43,10 +43,35 @@ A lock can target the following objects or attributes:
 
 To create a new lock, one can run the `tctl lock` command:
 
-```code
-$ tctl lock --user=foo@example.com --message="Suspicious activity." --ttl=10h
-# Created a lock with name "dc7cee9d-fe5e-4534-a90d-db770f0234a1".
-```
+<Tabs>
```

Comment:
Maybe add a bit more information to the first sentence of this section in order to reduce repetition and noise within the tabs? E.g. the first sentence could be, "To create a new lock on a Teleport resource, run one of the following `tctl lock` commands depending on the resource." Then the tabs could be "Username," "Role," "Multi-Factor Device ID," etc.